lib/puppet/provider/service/service.rb

```diff
@@ -36,6 +36,9 @@ def ucommand(type, fof = true)
     else
       cmd = [send("#{type}cmd")].flatten
     end
+    if (( Facter.value(:operatingsystem) == "Solaris" ) and ( type =~ /^start/ ))
```

Comment:
Is this the case only on Solaris, or could this happen on other Solaris derived platforms? (Illuminos, OpenIndiana, Nextenta, etc?)